.circleci/config.yml

```diff
@@ -46,6 +46,9 @@ jobs:
       - *add_ssh_keys
       - checkout
       - *restore_cache
+      - run:
+        name: Link packages
```

Comment:
looks like we've got a small typo. Also, the indentation is not the same as the other `run` commands. While it looks like this works, let's align it with the other ones. This allowance for variance brings up a profound philosophical question: WHO WILL LINT THE LINTER!